### ISSUE TITLE
[Rabobankapp name] compatibility report

### ISSUE THREAD

**Issue by itsmir** [AdAway/AdAway#2704]
**Describe the third party app**
 - Name: Rabobank
 - Version: [7.8.2]

**Describe the compabitily issue**
App opens but after pin code is entered nothing is displayed.

**Workaround**
No workaround - Whitelisting *rabo* does nothing only pausing adaway allows access

Phone Mi5s Lineage 16 (Android 9) rooted with Magisk
Adaway version 5.3.0 - in app update fails

**Comment by Japhys** [comment]
Ha, that's quite a coincidence. I have been running into that same problem with my OnePlus Nord / Lineageos 17 ever since I installed it. Only now bothered to open github to post, only to see the latest issue posted was about this  exact problem.

I usually just disable adaway when I need my banking app, but that's not a solution if course.

**Comment by Treinkapitein** [comment]
Also facing the same issue with the Rabo wallet-app
Quite awkward experience at the supermarket.
OnePlus 6, Lineage 17

**Comment by Japhys** [comment]
In 5.5.1 this issue had been solved, it seems

**Comment by Treinkapitein** [comment]
Updated to 5.5.1, confirmed working.

**Comment by Japhys** [comment]
Haha or maybe not, because adaway is not blocking anything anymore at the moment. @Treinkapitein is this new version still working for you

**Comment by casper2002casper** [comment]
Still have this issue with version 5.6.0 :/ (Lineage 18.1)

**Comment by Treinkapitein** [comment]
Updated to 5.6.0, works great so far.
Tested amazon and Rabobank

**Comment by itsmir** [comment]
Tested v5.6.0 (clean install not update) - problem still exists for me (Mi5s, lineage 16/A8)

**Comment by casper2002casper** [comment]
Got it working again by whitelisting "w.usabilla.com" which I noticed it making a call to on startup.

**Comment by itsmir** [comment]
Reinstalled v5.6.0 to test whitelisting per @casper2002casper  . Ran the log and attempted to access Rabobank app, to my surprise it's now working out of the box.  No need to whitelist anything (Mi5s, lineage 16)

**Comment by Japhys** [comment]
Got it working as well for a while. Latest version, w.usuability.com whitelisted. But didn't last long :)

**Comment by itsmir** [comment]
Yup - also stopped working here after a reboot.  Rolled back to  4.3.5.

**Comment by f3bruary** [comment]
I'm experiencing this as well on a Mi 11 Lite 5g

**Comment by WhatTheHai** [comment]
Doesn't work on 5.7.0, OnePlus 8P. Tried the suggested whitelists but no dice.

**Comment by itsmir** [comment]
After last update having the same issue on Xiaomi Mi9T - Global EEA rom (rooted).

**Comment by IkBenPatrick** [comment]
What resolved this for me are the following steps:

**Adaway app**
Select > hamburger menu (bottom left in the adaway app)
Select > Preferences
Select > Root based ad blocker
Disable/uncheck > Enable Web Server

Source/how I came across this: AdAway also had issues with the Prime Video app. These steps also resolved this problem. https://forum.xda-developers.com/t/adaway-causing-trouble-in-amazon-app.4197073/

**Comment by Japhys** [comment]
Thanks! This works indeed!

**Comment by f3bruary** [comment]
What are the disadvantages of disabling the web server ?

**Comment by Orteeltje** [comment]
> 
> 
> What are the disadvantages of disabling the web server ?

I would like to know to. 
Rooted Oneplus 6T with Oxygen 11.1.1.1. and Adaway 5.8.0.

Shouldn'd there be an solution in Adaway so the webserver can be enable?

**Comment by f3bruary** [comment]
This issue has become bigger since a recent Rabobank App update. Previously I was not able to view my transactions, but I could still use the app to pay via QR/Webhook. Now you can't do that either and the app closes when attempting to do so.

When AdAway is disabled everything works fine again.

I'm not sure why this is happening. I've recorded all the DNS requests and whitelisted the domains it's connecting to, but it just won't work.

**Comment by Kreator73** [comment]
Found a fix for this somewhere on the internet and it's working.  My Adaway is version 6.1.0 and I'm using the latest rabobank app. I'm on Android 13 

First of all
whitelist "w.usabilla.com" 
After that go to your hamburger menu (bottom left in the adaway app)
Select > Preferences
Select > Root based ad blocker
Disable/uncheck > Enable Web Server

And your done! It's letting you acces the rabobank app without pausing Adaway

**Comment by pal-ayaya** [comment]
@f3bruary because it's a webapp wrapped in a native app, and the chromium instance it used blindly calls `window.webAnalytics`

```
chromium    I  [INFO:CONSOLE(1)] "Uncaught TypeError: Cannot read properties of undefined (reading 'webAnalytics')", source:  (1)
chromium    I  [INFO:CONSOLE(1)] "Uncaught TypeError: Cannot read properties of undefined (reading 'webAnalytics')", source:  (1)
chromium    I  [INFO:CONSOLE(1)] "Uncaught TypeError: Cannot read properties of undefined (reading 'webAnalytics')", source:  (1)
chromium    I  [INFO:CONSOLE(1)] "Uncaught TypeError: Cannot read properties of undefined (reading 'webAnalytics')", source:  (1)
(...repeated 4 more times)
```

Recompiling it after removing these calls should fix the app.
